rpc/src/v1/impls/eth.rs

```diff
@@ -100,13 +99,13 @@ impl<C, S: ?Sized, M, EM> EthClient<C, S, M, EM> where
 		}
 	}
 
-	fn block(&self, id: BlockID, include_txs: bool) -> Result<Value, Error> {
+	fn block(&self, id: BlockID, include_txs: bool) -> Result<Option<Block>, Error> {
```

Comment:
What would it consist of? The errors that can occur still need to be serialized and won't be used anywhere else.